NEWS

```diff
@@ -251,7 +251,7 @@ CHANGES WITH 248:
           be restored for individual services with NoExecPaths=/dev (or by allow-
           listing and excluding /dev from ExecPaths=).
 
-        * Permissions for /dev/vsock are now set to 0o666, and /dev/vhost-vsock
+        * Permissions for /dev/vsock are now set to 0666, and /dev/vhost-vsock
```

Comment:
This is not a typo, it's the notation we use for octal

Reply:
Ah, have never seen anybody using this. I think the problem here is that `0666` (with leading `0`) is commonly used for suid, guid and sticky bit. How about using `0o0666`? That looks less like a typo IMHO.

Reply:
it's python syntax, which is derived from the C syntax for hex I guess. And quite frnakly it makes a lot of sense, i.e. the "0" prefix indicating octal is just a C stupidity if you ask me.

That said, access modes are established to be written in octal, hence I'd just write this as 0666 or 666 or so, it's better understood then. No need to teach people about Python syntax here.

or in other words, I think the suggested fix is actually OK.